packages/minifier-js/package.js

```diff
@@ -1,22 +1,17 @@
 Package.describe({
```

Comment:
I wouldn't recommend copying over my fork of minifier-js as is. I no longer use it, and it is outdated compared to the version in core.

Reply:
Yes, after discussing the best way to bring community packages to the core and implementing them as is, I will try to resolve the current issues present there and modernize the code.  It's a future job, but you will be included in the process for sure

Reply:
I created another and simpler approach https://github.com/meteor/meteor/pull/13715 bringing only the swc